File: app/src/main/java/com/skyd/imomoe/util/dlna/dms/IMediaContentDao.java
package com.skyd.imomoe.util.dlna.dms;

import android.content.Context;

import androidx.annotation.NonNull;

import org.fourthline.cling.support.model.item.Item;

import java.util.List;

interface IMediaContentDao {
    @NonNull
    List<Item> getImageItems(@NonNull Context context);

    @NonNull
    List<Item> getAudioItems(@NonNull Context context);

    @NonNull
    List<Item> getVideoItems(@NonNull Context context);
}

File: app/src/main/java/com/skyd/imomoe/util/dlna/dms/IMediaContentDao.kt
package com.skyd.imomoe.util.dlna.dms

import android.content.Context
import org.fourthline.cling.support.model.item.Item

internal interface IMediaContentDao {
    fun getImageItems(context: Context): List<Item?>
    fun getAudioItems(context: Context): List<Item?>
    fun getVideoItems(context: Context): List<Item?>
}